src/frontend/ForumIndex.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-
+import { Link } from 'react-router-dom';
 import './ForumIndex.css'
 
 export default class ForumIndex extends React.Component {
```

Comment:
* ThreadLink Component needs to be defined (#2), for example 
```
class ThreadLink extends React.Component  {
   render() {
    return (<div className='threadContainer'> <Link>blahblah</Link></div>);
   }
}
```
  * ThreadLink should have a state (eg. `this.state = { text: '' }`) and you should use that text value in its render method (eg `someothercode... <Link>{this.state.text}</Link> ...someothercode`)


  * ForumIndex Component needs to have, within its state, an array of ThreadLinks, eg. 
```
this.state = {
  threadLinks: [<ThreadLink />, <ThreadLink />, <ThreadLink />]
}
```